great_expectations/cli/datasource.py

```diff
@@ -319,7 +319,7 @@ def yaml_snippet(self) -> str:
     class_name: InferredAssetFilesystemDataConnector
     base_directory: {self.base_path}
     default_regex:
-      group_names: 
+      group_names:
```

Comment:
My editor likes to remove whitespace on save, apologies for the confusion!

Reply:
ah I think there were 2 tests that were failiing as a result of this. They've been updated